src/components/Vuetify/util/goTo.js

```diff
@@ -0,0 +1,92 @@
+import { consoleError } from '../../../util/console'
+
+const defaults = {
+  duration: 500,
+  offset: 0,
+  easing: 'easeInOutCubic'
+}
+
+const easingPatterns = {
+  // linear
+  linear: t => t,
+  // accelerating from zero velocity
+  easeInQuad: t => t * t,
+  // decelerating to zero velocity
+  easeOutQuad: t => t * (2 - t),
+  // acceleration until halfway, then deceleration
+  easeInOutQuad: t => (t < 0.5 ? 2 * t * t : -1 + (4 - 2 * t) * t),
+  // accelerating from zero velocity
+  easeInCubic: t => t * t * t,
+  // decelerating to zero velocity
+  easeOutCubic: t => --t * t * t + 1,
+  // acceleration until halfway, then deceleration
+  easeInOutCubic: t => t < 0.5 ? 4 * t * t * t : (t - 1) * (2 * t - 2) * (2 * t - 2) + 1,
+  // accelerating from zero velocity
+  easeInQuart: t => t * t * t * t,
+  // decelerating to zero velocity
+  easeOutQuart: t => 1 - --t * t * t * t,
+  // acceleration until halfway, then deceleration
+  easeInOutQuart: t => (t < 0.5 ? 8 * t * t * t * t : 1 - 8 * --t * t * t * t),
+  // accelerating from zero velocity
+  easeInQuint: t => t * t * t * t * t,
+  // decelerating to zero velocity
+  easeOutQuint: t => 1 + --t * t * t * t * t,
+  // acceleration until halfway, then deceleration
+  easeInOutQuint: t => t < 0.5 ? 16 * t * t * t * t * t : 1 + 16 * --t * t * t * t * t
+}
+
+function getDocumentHeight () {
+  return Math.max(
+    document.body.scrollHeight,
+    document.documentElement.scrollHeight,
+    document.body.offsetHeight,
+    document.documentElement.offsetHeight,
+    document.body.clientHeight,
+    document.documentElement.clientHeight
+  )
+}
+
+function getTargetLocation (target, settings) {
+  const documentHeight = getDocumentHeight()
```

Comment:
so it works only for scrolling the whole page? could it be extended to scroll a scrollable divs as well?

Reply:
So like accept another argument with the element we want to scroll? Or put it in options? Not sure how should this be done.

Reply:
Me too, I was just asking, if you think it will complicate things too much then skip it (we can always extend the functionality later)